app.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * App.js file that handles routing and basic error handling
+ * @type {*}
+ */
+var express = require('express');
+var bodyParser = require('body-parser');
+var errorTracker = require('./routes/errortracker');
+
+
+if(process.env.NODE_ENV === 'production'){
+  require('@google/cloud-trace').start();
```

Comment:
Since we discussed performance was a key issue for the tool, I thought it would come in handy.